projects/vdk-plugins/quickstart-vdk/cicd/release_acceptance_test_heartbeat_config.ini

```diff
@@ -0,0 +1,9 @@
+[DEFAULT]
+# Generate with https://console-stg.cloud.vmware.com/csp/gateway/portal/#/user/tokens
+# VDKCLI_OAUTH2_REFRESH_TOKEN= passed as environment variable
+VDKCLI_OAUTH2_URI=https://console-stg.cloud.vmware.com/csp/gateway/am/api/auth/api-tokens/authorize
+CONTROL_API_URL=http://cicd-control-service-svc:8092
```

Comment:
Should these links be hard-coded?

Reply:
Why not? Do you think they should not be public?

Reply:
I don't think it's about them being public. More like, what happens if the links change, that would require a code change, as well.
However, I don't think it's anything major, so you can ignore the comment and resolve the conversation.

Reply:
Yes, I see, you have a point. I checked and the URLs are hard-coded in .ini files frequently. I suppose they do not change often. Maybe, if we decide to use environment vars, we should do it in an effort that addressed all such places. I will check this and write a comment here if we decide to pursue such an effort. Thanks!